### ISSUE TITLE
Error : flash on Arduino Mega 2560 

### ISSUE THREAD

**Issue by NLGRF** [tinygo-org/tinygo#1340]
## Code :
```go
package main

import (
	"machine"
	"time"
)

func main() {
	led := machine.LED
	led.Configure(machine.PinConfig{Mode: machine.PinOutput})
	for {
		led.Low()
		time.Sleep(time.Millisecond * 1000)

		led.High()
		time.Sleep(time.Millisecond * 1000)
	}
}
```
## Using :
```cmd
tinygo flash -target arduino-mega2560 -port COM4 .
```

## I was getting:
```
avrdude: invalid file format '\Users\non19\AppData\Local\Temp\tinygo411124079\main.hex' in update specifier
avrdude: error parsing update operation 'flash:w:C:\Users\non19\AppData\Local\Temp\tinygo411124079\main.hex'
error: failed to flash C:\Users\non19\AppData\Local\Temp\tinygo411124079\main.hex: exit status 1
```

🔢 I try to flash on the Arduino UNO, Arduino Nano by arduino, arduino-nano. => `Success` 👍

**Comment by aykevl** [comment]
I think I see the problem.

Can you try editing the file `targets/arduino-mega2560.json`?

```diff
-    "flash-command":"avrdude -c wiring -b 115200 -p atmega2560 -P {port} -U flash:w:{hex} -v -D"
+    "flash-command":"avrdude -c wiring -b 115200 -p atmega2560 -P {port} -U flash:w:{hex}:i -v -D"
```

The difference is the extra `:i` at the end.

**Comment by aykevl** [comment]
See: https://github.com/tinygo-org/tinygo/pull/1341

**Comment by NLGRF** [comment]
> See: #1341



> I think I see the problem.
> 
> Can you try editing the file `targets/arduino-mega2560.json`?
> 
> ```diff
> -    "flash-command":"avrdude -c wiring -b 115200 -p atmega2560 -P {port} -U flash:w:{hex} -v -D"
> +    "flash-command":"avrdude -c wiring -b 115200 -p atmega2560 -P {port} -U flash:w:{hex}:i -v -D"
> ```
> 
> The difference is the extra `:i` at the end.

@aykevl Thanks

**Comment by NLGRF** [comment]
> I think I see the problem.
> 
> Can you try editing the file `targets/arduino-mega2560.json`?
> 
> ```diff
> -    "flash-command":"avrdude -c wiring -b 115200 -p atmega2560 -P {port} -U flash:w:{hex} -v -D"
> +    "flash-command":"avrdude -c wiring -b 115200 -p atmega2560 -P {port} -U flash:w:{hex}:i -v -D"
> ```
> 
> The difference is the extra `:i` at the end.

what is `:i` , stands for?

**Comment by aykevl** [comment]
> what is `:i` , stands for?

Intel hex. This is the default. However, on Windows paths often contain a colon (`:`) and so the rest of the path is interpreted as a format, which you can see in the error message:

    avrdude: invalid file format '\Users\non19\AppData\Local\Temp\tinygo411124079\main.hex' in update specifier

"Invalid file format refers to everything behind `C:`, which is interpreted as the file format.

The same issue happened before, see #851.

**Comment by deadprogram** [comment]
This issue was corrected in v0.15.0 which was just released. Now closing.
